domain_access/domain_access.module

```diff
@@ -519,6 +525,8 @@ function domain_access_views_data_alter(array &$data) {
 
 /**
  * Implements hook_ENTITY_TYPE_insert().
+ *
```

Comment:
hook implementation params should not need documenting as per https://www.drupal.org/node/1354#hookimpl